src/test/user/user-request-authorizer.spec.ts

```diff
@@ -96,5 +101,30 @@ describe("UserRequestAuthorizer", () => {
         })
         .catch(() => done(new Error("Promise should have been resolved")));
     });
+
+    it("should resolve if the user has an authorized role", () => {
+      return Promise.resolve(userRequestAuthorizer.authorise(request))
+        .then((result) => {
+          expect(roleAuthorizer.isUserAuthorized).to.have.been.calledWith(DETAILS);
+          expect(result).to.equal(DETAILS);
+        })
+        .catch(() => {
+          throw new Error("Promise should have been resolved");
+        });
+    });
+
+    it("should reject if the user has no authorized roles", (done) => {
```

Comment:
This one uses `authorized` :)

Reply:
Now made consistent with American spelling